src/python/nimbusml/internal/core/preprocessing/schema/columnselector.py

```diff
@@ -65,8 +83,15 @@ def _get_node(self, **all_args):
                 "input has to be a list of strings, instead got %s" %
                 type(input_columns))
 
+        keep_columns = self.keep_columns
+        if self.keep_columns is None and self.drop_columns is None:
```

Comment:
I made an issue discussing how to deal with the change in the ML.NET entrypoint: https://github.com/Microsoft/NimbusML/issues/76

Reply:
Nice fix @ 

---
In reply to: [243136165](https://github.com/Microsoft/NimbusML/pull/55#discussion_r243136165) [](ancestors = 243136165)